apps/sms/js/utils.js

```diff
@@ -653,6 +653,7 @@
           canvas = Utils.imageToCanvas(
             img, adjustedSize.width, adjustedSize.height
           );
+          img.src = '';
```

Comment:
Okay, if the root cause of this issue is here then I'm fine, just try to call ```cleanup``` here instead and remove it from ```deferred.promise.then``` callback.